mobile/src/components/setting/setting.tsx

```diff
@@ -47,6 +44,10 @@ const Settings: FC = () => {
     dataStatus: userDetails.dataStatus,
   }));
 
+  const GENDER_OPTIONS = Object.values(UserGender).map((gender) => ({
```

Comment:
I don't like this solution, maybe the problem that dropdown tries to modify items? Try to pass copy of the constant